bridge-offchain-worker/README.md

```diff
@@ -78,7 +78,7 @@ Save and edit the following to a `.env` file inside of `bridge-offchain-worker/`
 SOURCE_CHAIN_BLOCK_LAG=3
 
 # Address of the ERC20 token contract on source chain
-SOURCE_CHAIN_CONTRACT_ADDRESS=<Addr of source chain contract you deployed in custom-contracts-bridging tutorial>
```

Comment:
As we discussed, reusing the hello-bridge contract addresses here could interfere with other learner's ability to complete that tutorial.